app/views/components/docs/contents-list.yml

```diff
@@ -60,7 +71,7 @@ examples:
       contents:
         - href: "#first-thing"
           text: 1. First thing
-        - href: "#second-thing"
+        - href:
```

Comment:
I'm probably lacking some context around exactly how this component will be used on guides and the kind of data we're getting back to feed into the component, but would it make more sense to support no href being passed in at all, rather than having to pass a blank/nil value?